package.json

```diff
@@ -1,19 +1,13 @@
 {
   "name": "@aserto/console-type-definitions",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
The current active tag is [1.5.0](https://github.com/aserto-dev/console-type-definitions/tags) so we should probably bump this to `1.6.0`